### PR TITLE
Update Time Picker example with Material specs

### PR DESCRIPTION
**Pull request by peluprvi**
Specs:
https://material.io/guidelines/components/pickers.html#pickers-time-pickers

### CODE REVIEW

File: examples/pickers/6.vue [hunk 1]
```diff
@@ -35,8 +35,9 @@
         <v-time-picker v-model="e6" actions>
           <template scope="{ save, cancel }">
             <v-card-actions>
+              <v-spacer></v-spacer>
               <v-btn flat primary @click.native="cancel()">Cancel</v-btn>
```
Comment (johnleider) [inline examples/pickers/6.vue]: Since you are already updating this, can you remove the **.native** from both buttons? It is no longer needed, thank you.

Comment (peluprvi) [reply examples/pickers/6.vue]: Of course! After merging that, do you think I can check and rename all example files to be matched with the documentation examples' numbers?

Comment (johnleider) [reply examples/pickers/6.vue]: One of the things we were tossing around was to name the examples something semantic, as opposed to a number. Examples can be seen here https://github.com/vuetifyjs/docs/tree/dev/examples/tables

Comment (peluprvi) [reply examples/pickers/6.vue]: Better approach. I think can help with that!

**Pull request merged**